data/governments.txt

```diff
@@ -677,6 +678,77 @@ government "Independent (Killable)"
 	"friendly hail" "friendly civilian"
 	"hostile hail" "hostile civilian"
 
+# These are Korath- and pirate-friendly humans that dislike the Syndicate
+government "Independent (Korath-Allied)"
+	swizzle 9
+	color "governments: Independent"
+	"display name" "Independent"
+
+	"player reputation" 10
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+	"attitude toward"
+		"Korath (Civilian)" .01
+
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+
+government "Independent (Korath-Allied) Port"
+	swizzle 9
+	color "governments: Independent"
+	"display name" "Independent"
+
+	"player reputation" 10
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+	"attitude toward"
+		"Syndicate" -.01
+		"Syndicate (Extremist)" -.01
+		"Republic" -.01
+		"Republic that won't enter wormhole" -.01
+		"Navy Intelligence" -.01
+		"Republic Intelligence" -.01
+		"Independent (Korath-Allied)" 1
+		# Negative for Korath ensures the Korath raiding groups will not come
+		# from Independent (Korath-Allied) Port planets and stations.
+		"Korath" -.01
+		"Kor Alaji" -.01
+		"Kor Isaha" -.01
+
+# Used for smuggling sites and other restricted-access stations. Allows Korath to land.
+government "Independent (Korath-Allied) Restricted"
+	swizzle 9
+	color "governments: Independent"
+	"display name" "Independent"
+
+	"player reputation" 10
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+	"attitude toward"
+		"Syndicate" -.01
+		"Syndicate (Extremist)" -.01
```

Comment:
```suggestion
```

Reply:
There is a corner case where when the Pug invade in Free Worlds Reconciliation, you go to Algenib, which has naturally spawning Syndicate (Extremist) fleets, then lure them all the way to Misam/Atik, and my suggested change will allow them to land. I am hiding the above comment as a result.